### PR TITLE
Csghub  collection add repo fix

### PR DESCRIPTION
**Pull request by ymh6315431**

### CODE REVIEW

File: app/javascript/components/collections/CollectionsAddRepo.vue [hunk 1]
```diff
@@ -138,16 +123,19 @@
...
+  const typeChange = () => {
     repoIdsInput.value = ''
-    fetchRepoList(type)
   }
-
-  const fetchRepoList = async (type) => {
-    const url = `/${type}`
+  const repoInputChange = (search) => {
+    repoIdsInput.value = ''
+    fetchRepoList(typeInput.value,search)
```
Comment (hiveer) [inline app/javascript/components/collections/CollectionsAddRepo.vue:134]: space 
```suggestion
    fetchRepoList(typeInput.value, search)
```

File: app/javascript/components/collections/CollectionsAddRepo.vue [hunk 2]
```diff
@@ -138,16 +123,19 @@
...
+  }
+  const fetchRepoList = async (type,search) => {
```
Comment (hiveer) [inline app/javascript/components/collections/CollectionsAddRepo.vue:136]: space
```suggestion
  const fetchRepoList = async (type, search) => {
```

File: app/javascript/components/collections/CollectionsAddRepo.vue (hunk 1 above)
Comment (ymh6315431) [reply app/javascript/components/collections/CollectionsAddRepo.vue]: done

File: app/javascript/components/collections/CollectionsAddRepo.vue (hunk 2 above)
Comment (ymh6315431) [reply app/javascript/components/collections/CollectionsAddRepo.vue]: done

**Review by hiveer — APPROVED** [review]
LGTM

**Pull request merged**